src/test/e2e/13_find_images_test.go

```diff
@@ -0,0 +1,61 @@
+package test
+
+import (
+	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestFindImages(t *testing.T) {
+	t.Log("E2E: Find Images")
+
+	t.Run("zarf test find images success", func(t *testing.T) {
+		t.Log("E2E: Test Find Images")
+
+		testPackagePath := filepath.Join("examples", "dos-games")
+		expectedOutput := []byte{}
+		f, err := os.Open("src/test/packages/13-find-images/dos-games-find-images-expected.txt")
+		defer f.Close()
+
+		_, err = f.Read(expectedOutput)
```

Comment:
`expectedOutput` isn't being populated with data from the txt file, so the test is passing because stdout does contain an empty string. We can change this to use `os.ReadFile` and capture the returned `[]bytes` as `expectedOutput` in each test to ensure the data is read from the files correctly

Reply:
so I was under the wrong impression that the tests were passing as the expectedOutput was being populated 🤦